scripts/fortran_unit_testing/run_tests.py

```diff
@@ -405,7 +405,9 @@ def _main():
             if ctest_args is not None:
                 ctest_command.extend(ctest_args.split(" "))
 
-            run_cmd_no_fail(" ".join(ctest_command), from_dir=label, combine_output=True)
+            logging.info("Running '{}'".format(" ".join(ctest_command)))
```

Comment:
This should be logger, not logging - look at line 25.

Reply:
Good catch, thanks.